llvm/lib/CodeGen/AsmPrinter/DwarfFile.cpp

```diff
@@ -94,7 +125,43 @@ unsigned DwarfFile::computeSizeAndOffset(DIE &Die, unsigned Offset) {
                                       Offset);
 }
 
-void DwarfFile::emitAbbrevs(MCSection *Section) { Abbrevs.Emit(Asm, Section); }
+void DwarfFile::emitAbbrevs(MCSection *Section) {
+  // If the Abbreviation vector only has one Abbreviation contribution and the
+  // contribution has non-zero Abbreviations in use, emit it without emitting
+  // any labels. This is done because there are some targets does not support
```

Comment:
Some targets do not support labels in debug info sections.

Reply:
I don't know why I messed up the grammar there 🤦🏾‍♂️